### PR TITLE
Update service-endpoints.md

### PR DESCRIPTION
**Pull request by ashokirla**
Added info about the changes to the security mode for service connections and cross project sharing feature.

### CODE REVIEW

**Comment by PRMerger17** [comment]
@ashokirla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@ashokirla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/library/service-endpoints.md [hunk 1]
```diff
@@ -66,53 +66,94 @@ Service connections are created at project scope. A service connection created i
...
    Or, in TFS, open the **Services** page from the "settings" icon in the top menu bar.
 
 1. Select the service connection you want to manage.
 
-1. Choose from the list of **Actions** in the **Details** tab in the right pane.
-
-The actions available depend on the chosen type of connection. You can update only
-some properties of connections; for example, to change the selected subscription
-you must re-create the connection. Choose **Disconnect** to delete or remove a connection.
+1. You will land in the **Overview** tab of the service connection where you can see the details of the service connection i.e. type, creator, authentication schema etc.
```
Comment (RoopeshNair) [inline docs/pipelines/library/service-endpoints.md]: authentication schema  - user may not relate to this

Comment (ashokirla) [reply docs/pipelines/library/service-endpoints.md]: Updated as suggested

**Comment by PRMerger10** [comment]
@ashokirla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@ashokirla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@ashokirla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@ashokirla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@ashokirla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@ashokirla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@ashokirla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@ashokirla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@ashokirla : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by RoopeshNair** [comment]
#sign-off

**Comment by ktoliver** [comment]
Hi @ashokirla 

Microsoft authors and contributors should work in the private repository, azure-devops-docs-pr. The changes to the article are extensive and should be verified on staging and checked by PRMerger validation before the content is merged. 

Could you close this PR and resubmit the approved version of your updates in a new PR in the private repo?

Thanks. If the PR remains open on 1/9/2020 (after three days), a PR reviewer will move the commit to the private repo. #hold-off

**Comment by ktoliver** [comment]
PR https://github.com/MicrosoftDocs/azure-devops-docs-pr/pull/784 in the private repository replaces this PR. We moved the commit to the private repo so the content could be validated and staged. We'll close this PR now.

**Pull request closed without merging**